babel.config.cjs

```diff
@@ -31,5 +31,19 @@ module.exports = {
         "@babel/plugin-transform-object-rest-spread",
         "@babel/plugin-syntax-dynamic-import",
         "@babel/plugin-transform-runtime",
+        [
+            "search-and-replace",
+            {
+                rules:
+                    process.env.NODE_ENV === "test"
```

Comment:
Why only "test"?

Reply:
For the same reason as [develop/src/matrix.ts#L133](https://github.com/matrix-org/matrix-js-sdk/blob/develop/src/matrix.ts?rgh-link-date=2024-09-06T13%3A56%3A21Z#L133), I assume, though this certainly needs clarifying with a code comment.

Reply:
(also, I'd find it easier to grok if the condition were reversed: a list of substitutions to perform when `process.env.NODE_ENV !== "test"`)

Reply:
Alright, I've cleaned that up.